tests/backup/backup_helper.go

```diff
@@ -27,7 +27,6 @@ const (
 	cloudAccountDeleteTimeout                 = 10 * time.Minute
 	cloudAccountDeleteRetryTime               = 30 * time.Second
 	storkDeploymentNamespace                  = "kube-system"
-	clusterName                               = "tp-cluster"
```

Comment:
Was this removed because it was not being used by function?

Reply:
yes

Reply:
done